README.md

```diff
@@ -10,7 +10,7 @@ The main software that is added to the non-NDK versions:
 
 The sdkmanager packages that are installed are generally the current version and the previous version.  Check the Dockerfile for more details.
 
-**Note:** From February 7th 2024, the build images are running as the unprivileged `gradle` user instead of `root`, if you encounter a build error saying "_Failed to install the following SDK components: ..._" then it means that the image is missing a component required by your Android build; send a PR to add this dependency or select another another container image.
+**Note:** From February 7th 2024, the build images are running as the unprivileged `gradle` user instead of `root`, if you encounter permission problems due to this change, send a PR to add the dependency you need or select a image variant that already have it included.
```

Comment:
We do not want to support PRs for multiple image variants.  The image must supporting adding additonal build components upon request.  I beleive your PR is doing that.  Can we delete this comment?